lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+            },
+            "actor": {
+              "reference": "Practitioner/11817978"
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Need notes around supported system and values for function</li>
+        <li>Both actor and function are required</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer
+
+  - name: reason
+    required: 'No'
+    type: CodeableConcept
+    description: Diagnosis codes for why the charge was rendered.
+    action: modify
+    example: |
+      {
+        "reason": [
+          {
+            "extension": [
+              {
+                "valueUnsignedInt": 1,
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+              }
+            ],
+            "coding": [
+              {
+                "extension": [
+                  {
+                    "valueCoding": {
+                      "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                      "code": "3693"
+                    },
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                  }
+                ],
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems.</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+
+  - name: Replacing Extension
+    required: 'Yes'
+    type: Extension
+    description: The resource that this ChargeItem is replacing.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueId": "1",
+            "url": "item-version"
+          }
+        ],
+        "valueReference": {
+          "reference": "ChargeItem/317370800"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: performing-location
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: modify
+    example: |
+      {
+        "valueReference": {
+          "reference": "Location/21304903",
+          "display": "NM"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: description
+    required: 'No'
+    type: Extension
+    description: A description providing additional details of the resource.
```

Comment:
Should this read something along the lines of `A description providing additional details about the ChargeItem.` ?

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R158-R161